client/elife-theme/src/index.js

```diff
@@ -41,12 +42,12 @@ export default {
 
   /* Spacing */
   gridUnit: '6px',
-  verticalXxl: 'gridUnit'*20,
-  verticalXl: 'gridUnit'*12,
-  verticalL: 'gridUnit'*8,
-  verticalM: 'gridUnit'*6,
-  verticalS: 'gridUnit'*4,
-  verticalXs: 'gridUnit'*2,
+  verticalXxl: 'gridUnit' * 20,
```

Comment:
Funny how commitizen fixed the spacing for something that's wrong in the first place anyway :)

Reply:
This is `prettier` and it only cares about syntax, not semantics.

Reply:
I meant prettier and I'm aware it only checks syntax, I was just saying it's funny.